riotfile.py

```diff
@@ -158,12 +162,17 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             ],
         ),
         Venv(
-            pys=["3"],
+            pys=["3.9"],
             pkgs={"slotscheck": latest},
             venvs=[
                 Venv(
                     name="slotscheck",
                     command="python -m slotscheck -v {cmdargs}",
+                    pkgs={
```

Comment:
are these needed?

Reply:
Without `pkgs`, slotscheck raise an error with pre-commit/hooks:
```
>> riot run slotscheck -v 'ddtrace/appsec/processor.py' 'ddtrace/contrib/flask/patch.py' 'ddtrace/contrib/trace_utils.py' 'tests/appsec/test_processor.py' 'tests/contrib/django/test_django_appsec.py' 'tests/contrib/flask/test_flask_appsec.py'

ERROR: Couldn't inspect module 'tests.contrib.flask.test_flask_appsec' due to ImportError("Error while finding loader for 'tests.contrib.flask.test_flask_appsec' (<class 'ModuleNotFoundError'>: No module named 'flask')"). Run `import tests.contrib.flask.test_flask_appsec` to reproduce this error.
```

Reply:
cc @majorgreys is this a bug?